parser/SqlParser.ypp

```diff
@@ -1332,6 +1369,83 @@ opt_limit_clause:
     }
   }
 
+opt_window_clause:
+  {
+    $$ = nullptr;
+  }
+  | window_declaration_list
+
+window_declaration_list:
+  window_declaration {
+    $$ = new quickstep::PtrList<quickstep::ParseWindow>();
+    $$->push_back($1);
+  }
+  | window_declaration_list window_declaration {
+    $$ = $1;
+    $$->push_back($2);
+  }
+
+window_declaration:
```

Comment:
@zuyu Since window_declaration is a part of window_declaration_list,I put it this way in order to maintain the hierarchy. If alphabet order has higher precedence then I will change them. Thanks!